cmd/create/operatorroles/cmd.go

```diff
@@ -164,12 +173,29 @@ func run(cmd *cobra.Command, argv []string) {
 		os.Exit(0)
 	}
 
-	prefix, err := aws.GetPrefixFromAccountRole(cluster)
+	prefix := args.prefix
```

Comment:
@pvasant - I think we should remove all `run` code for this flag. We should only add it so that any automation scripts do not fail if they set the flag, but the flag itself should be a no-op.

@arendej - Does this make sense?

Reply:
@vkareh  there is really none.  It only has the validation and then I m overriding with this. Let me know if you want me to remove these validations also and just neglect the parameter supplied
```

prefixUsed, err := aws.GetPrefixFromAccountRole(cluster)
	if err != nil {
		reporter.Errorf("Failed to find prefix from %s account role", aws.InstallerAccountRole)
		os.Exit(1)
	}

	if prefix != "" && prefix != prefixUsed {
		reporter.Errorf("Operator Prefix '%s' mismatches with prefix '%s' from %s account role. Please check "+
			"and try again", prefix, prefixUsed, aws.InstallerAccountRole)
		os.Exit(1)
	}
	prefix = prefixUsed








```

Reply:
Yes, the goal is to _not_ use the prefix any more. We don't want the user entering a prefix in this command since we already have all the information we need. We should accept the arg in the command line, send a deprecated warning, and then ignore the flag entirely.